CMakeLists.txt

```diff
@@ -67,6 +67,8 @@ endif()
 option(LLAMA_ACCELERATE             "llama: enable Accelerate framework"                    ON)
 option(LLAMA_OPENBLAS               "llama: use OpenBLAS"                                   OFF)
 option(LLAMA_CUBLAS                 "llama: use cuBLAS"                                     OFF)
+option(LLAMA_CUDA_BY                "llama: y block size for dmmv CUDA kernels"             1)
```

Comment:
If not set manually by the user the value 1 seems to be converted to "ON", thus causing a compile error. If you manually set it to an integer it works. I'm a total noob when it comes to cmake, does someone know how to fix this?

Reply:
`option()` is boolean.
It needs to be something like:
```cmake
set(LLAMA_CUDA_BY "1" CACHE STRING "llama: y block size for dmmv CUDA kernels")
```

Reply:
That seems to have fixed it, thanks.